### PR TITLE
Add CII Best Practices badge to README.md

### PR DESCRIPTION
**Pull request by eqrx**
**What this PR does / why we need it**:
I would like to add the CII Best Practices badge to KubeOne since it stands for a degree of code quality. See https://bestpractices.coreinfrastructure.org for more info and [CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2934) for the KubeOne badge

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
None

**Special notes for your reviewer**:
Please verify that the information I entered into the form is correct. Mostly its "Where do you find X" but some query for details that I am not fully sure about. This is mostly the case in the "Analysis" tab

**Documentation**:
See above

**Release note**:
```release-note
KubeOne is now CII Best Practices certified
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
   <a href="https://goreportcard.com/report/github.com/kubermatic/kubeone">
     <img src="https://goreportcard.com/badge/github.com/kubermatic/kubeone" alt="Go Report Card" />
   </a>
+  <a href="https://bestpractices.coreinfrastructure.org/projects/1189"><img src="https://bestpractices.coreinfrastructure.org/projects/1189/badge">
```
Comment (xmudrii) [inline README.md]: This should link to the KubeOne page, not to the Fluentd page.

Comment (eqrx) [reply README.md]: Sorry, fixed

**Pull request merged**